README.md

```diff
@@ -123,3 +123,22 @@ two json files have been loaded, and the related checkbox will enable/disable th
 
 The functionalities for the graph drawing and its behavior are developed in javascript and are avaialble
 at the following [repository](https://github.com/oda-hub/renku-aqs-graph-library/).
+
+
+# Installation of the plugin
+
+Currently, the plugin is developed using the version `2.1.0` of [renku-python](https://github.com/SwissDataScienceCenter/renku-python). Please make sure such version is installed by running:
+
+```bash
+renku --version
+```
+
+Despite the dependency is guaranteed during the installation if the plugin, and additional check will be performed during the import.
```

Comment:
I am not sure this is a complete sentence. Despite => what? :)

Reply:
Yeah it made no sense...